### PR TITLE
[BT-1823] match dataxu common python project structure

### PR DESCRIPTION
**Pull request by careyrp**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+requests == 2.10
```
Comment (dmepham) [inline requirements.txt]: I think this file is redundant if we're creating a wheel with setup.py and the requirements are already listed in `setup_requires`. see https://github.com/dataxu/jira-webhook/blob/master/requirements.txt

File: .python-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.7.10
```
Comment (dmepham) [inline .python-version]: what consumes this?

**Comment by dmepham** [comment]
looks good to me

File: .python-version (hunk 2 above)
Comment (careyrp) [inline .python-version]: pyenv

**Pull request merged**